doc/source/create-k8s-cluster.rst

```diff
@@ -163,6 +163,98 @@ Setting up Kubernetes on Microsoft Azure Container Service (ACS)
 
    The response should list three running nodes.
 
+.. _amazon-aws:
+
+Setting up Kubernetes on Amazon Web Services (AWS)
+--------------------------------------------------
+
+AWS does not have native support for Kubernetes, however there are
+many organizations that have put together their own solutions and
+guides for setting up Kubernetes on AWS. We like the `guide put
+together by Heptio <https://s3.amazonaws.com/quickstart-reference/heptio/latest/doc/heptio-kubernetes-on-the-aws-cloud.pdf>`_,
+and recommend using this for setting up your stack.
+
+1. Follow Step 1 of the Heptio guide. This sets up your Amazon account with
+   the credentials needed to run Kubernetes.
+
+   .. note::
+
+      Make sure that you keep the file downloaded when you create the SSH
+      key. This will be needed later to allow ``kubectl`` to interact with
+      your Kubernetes cluster.
+
+   .. note::
+
+      You may find it helpful to "pin" the services we'll be using to your AWS
+      navbar. This makes it easier to navigate in subsequent sessions.
+      Click the "pin" icon at the top, then drag ``CloudFormation`` and
+      ``EC2`` into your navbar.
```

Comment:
I feel like there are other places that deserve more screenshot real-estate than here. Since we don't use screenshots anywhere else, maybe instead of doing it in this PR we should discuss other locations where this would be maximally helpful? WDYT?